docs/releases/1.10.txt

```diff
@@ -385,6 +385,30 @@ Dropped support for PostgreSQL 9.1
 Upstream support for PostgreSQL 9.1 ends in September 2016. As a consequence,
 Django 1.10 sets PostgreSQL 9.2 as the minimum version it officially supports.
 
+.. _default-logging-changes-110:
+
+Changes to the default logging configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``runserver`` command now logs each request to the
+:ref:`django-server-logger` logger instead of outputting it directly to console;
+if your logging config overrides Django's defaults, see the :ref:`Django's
+default logging configuration <default-logging-configuration>` for an example of
+how to configure runserver logging with color output.
+
+To make it easier to write custom logging configurations, Django's default
+logging configuration now defines the 'django.server' logger, filtered at the
+``INFO`` level, with a single handler:
+
+* 'django.server': filtered at the ``INFO`` level and only active if
+``DEBUG=True``.
+
+If you aren't overriding Django's default logging, you should see minimal
```

Comment:
These sentences aren't very useful. Again, the main thing to answer is if I disable Django's logging, how do I modify my configuration so that runserver output is restored.